cylc/flow/parsec/jinja2support.py

```diff
@@ -182,6 +182,14 @@ def jinja2environment(dir_=None):
     env.globals['environ'] = os.environ
     env.globals['raise'] = raise_helper
     env.globals['assert'] = assert_helper
+
+    # if template_vars is not None:
+    #     # I expect multiple things in this block
+    #     if 'ROSE_SUITE_VARIABLES' in template_vars:
+    #         env.globals['ROSE_SUITE_VARIABLES'] = template_vars[
```

Comment:
Ahha, @wxtim, this was the code I was questioning. Although uncommenting does not help with the test failures.

Reply:
Yes, I ought to uncomment it.